docs/ispc.rst

```diff
@@ -2432,6 +2432,13 @@ Variables can also be declared in ``for`` statement initializers:
 
     for (int i = 0; ...)
 
+Varying variables can be initialized with individual element values in braces.
+The number of values has to be equal to the target width:
```

Comment:
Please mention that using static varying initialization is not portable across targets with different width. The recommended way to deal with it is to guard it with specific TARGET_WIDTH ifdef. I think the example should use use as well.

Reply:
Please add description of TARGET_WIDTH and TARGET_ELEMENT_WIDTH to the table in line 846.